aiven/client/cli.py

```diff
@@ -64,7 +64,13 @@ def convert_str_to_value(schema: Mapping[str, Any], str_value: Optional[str]) ->
                     "Invalid boolean value {!r}: expected one of {}".format(str_value, ", ".join(values))
                 ) from ex
         elif "array" in schema["type"]:
-            return [convert_str_to_value(schema["items"], val) for val in str_value.split(",")]
+            str_value_array = []
```

Comment:
This seems unnecessary, we can do `str_value_array = [jsonlib.loads(str_value)]` on line 70 for the same end result.

Reply:
yes but if we break it up and have a `return` on line 70 and line 72 `mypy` complains about having too many `return` statements. Unless there is another way of approaching it without having two `return` statements.